src/comments/comments.service.ts

```diff
@@ -29,17 +29,9 @@ export class CommentsService {
     if (!result) {
       throw new Error('Could not add comment');
```

Comment:
Use a nest exception here instead of a plain error

Reply:
NotFoundException?